src/v3/src/components/PhoneAuthenticator/PhoneAuthenticator.tsx

```diff
@@ -133,82 +122,38 @@ const PhoneAuthenticator: UISchemaElementComponent<UISchemaElementComponentWithV
   const renderExtension = () => (
     showExtension && (
       <Box width={0.25}>
-        <InputLabel
-          htmlFor="phoneExtension"
-          // label should remain in rtl format if rtl language is set
-          dir={languageDirection}
-        >
-          {extensionLabel}
-        </InputLabel>
-        <InputBase
-          value={extension}
-          type="text"
-          name="extension"
+        <TextField
+          autoCompleteType={disableAutocomplete ? 'off' : 'tel-extension'}
           id="phoneExtension"
-          dir="ltr"
-          disabled={loading}
+          isDisabled={loading}
+          label={extensionLabel ?? ''}
+          name="extension"
           onChange={(e: ChangeEvent<HTMLInputElement | HTMLTextAreaElement>) => {
             setExtension(e.currentTarget.value);
           }}
-          inputProps={{
-            'data-se': 'extension',
-            autocomplete: disableAutocomplete ? 'off' : 'tel-extension',
-            'aria-describedby': ariaDescribedBy,
-          }}
+          testId="extension"
+          value={extension}
         />
       </Box>
     )
   );
 
   const renderCountrySelect = () => (
     <Box marginBlockEnd={4}>
-      <InputLabel
-        id="countryLabel"
-        htmlFor="country"
-        // To prevent asterisk from shifting far right
-        sx={{ justifyContent: showAsterisk ? 'flex-start' : undefined }}
-      >
-        {countryLabel}
-        {showAsterisk && (
-          <Box
-            component="span"
-            sx={{
-              marginInlineStart: tokens.Spacing2,
-              marginInlineEnd: tokens.Spacing2,
-            }}
-            className="no-translate"
-            aria-hidden
-          >
-            *
-          </Box>
-        )}
-        {required === false && (
-          <Typography
-            variant="subtitle1"
-            sx={{ whiteSpace: 'nowrap' }}
-          >
-            {optionalLabel}
-          </Typography>
-        )}
-      </InputLabel>
-      <Select
+      <NativeSelect
+        autoCompleteType={disableAutocomplete ? 'off' : 'tel-country-code'}
+        hint={!required ? optionalLabel : undefined}
```

Comment:
Move the optional label to the Odyssey `hint` prop, this is also done in `Select` migration